heaps/min_heap.py

```diff
@@ -21,21 +19,35 @@ def __init__(self):
     def add(self, key, value = None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(log n)
+            Space Complexity: O(1)
         """
```

Comment:
👍 However this is O(log n) for space complexity because `heap_up` is recursive and you have the call stack.